### PR TITLE
[2.6] Add istio 1.12.6

### PR DESCRIPTION
**Pull request by doflamingo721**
Add support for istio 1.12.6:
https://istio.io/latest/news/releases/1.12.x/announcing-1.12.6/

Linked issue:
https://github.com/rancher/rancher/issues/37218

Adds:
kiali-server:1.44.0
jaeger-tracing:1.32.0
istio:1.12.6

### CODE REVIEW

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
Changes overall look good, but rebase is required and approval is pending the merge of https://github.com/rancher/istio-installer/pull/52 and https://github.com/rancher/image-mirror/pull/231, which have already been approved.

Once approved and rebased, reach out to me for a final review and I can approve this.

**Comment by doflamingo721** [comment]
@aiyengar2 I've rebased the PR and the respective image-mirror PR https://github.com/rancher/image-mirror/pull/231 and istio-installer PR https://github.com/rancher/istio-installer/pull/52 have been merged. Can you please review it once?

File: packages/rancher-istio/rancher-istio/charts/values.yaml
Comment (aiyengar2) [inline packages/rancher-istio/rancher-istio/charts/values.yaml:49]: https://hub.docker.com/r/rancher/mirrored-istio-pilot/tags?page=1&name=1.12.6

File: packages/rancher-istio/rancher-istio/charts/values.yaml
Comment (aiyengar2) [inline packages/rancher-istio/rancher-istio/charts/values.yaml:61]: image exists https://hub.docker.com/r/rancher/mirrored-istio-proxyv2/tags?page=1&name=1.12.6

**Pull request merged**
